kafka/stackable/patches/3.7.1/001-cyclonedx-plugin.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/build.gradle b/build.gradle
+index 32e6e8f..d496382 100644
+--- a/build.gradle
++++ b/build.gradle
+@@ -48,6 +48,45 @@ plugins {
+   // artifacts - see https://github.com/johnrengelman/shadow/issues/901
+   id 'com.github.johnrengelman.shadow' version '8.1.0' apply false
+   id 'com.diffplug.spotless' version '6.14.0' apply false // 6.14.1 and newer require Java 11 at compile time, so we can't upgrade until AK 4.0
++  id 'org.cyclonedx.bom' version '1.9.0'
++}
++
++cyclonedxBom {
++    // Specified the type of project being built. Defaults to 'library'
++    projectType = "application"
++    // Specified the version of the CycloneDX specification to use. Defaults to '1.5'
++    schemaVersion = "1.5"
++    // Boms destination directory. Defaults to 'build/reports'
++    destination = file("build/reports")
++    // The file name for the generated BOMs (before the file format suffix). Defaults to 'bom'
++    outputName = "bom"
++    // The file format generated, can be xml, json or all for generating both. Defaults to 'all'
++    outputFormat = "json"
++    includeConfigs = ["runtimeClasspath"]
++    skipProjects = [
```

Comment:
Why are exactly these projects skipped?

Reply:
They added some vulnerabilities that popped up when scanning the SBOM, which where not there when I scanned the container image.
I also did not find these projects in the container image and from the `build.gradle` spec it also looks like these are test dependencies which are not shipped by us.

Reply:
A comment could be added that this list can be retrieved with `gradle projects | grep upgrade-system-tests`. This makes it clear what to do when adding a patch for a new Kafka version.

Reply:
Makes sense, I added a comment.